### ISSUE TITLE
Base example doesnt work?

### ISSUE THREAD

**Issue by imbesci** [jxnl/instructor#159]
Hi jason, watched your Pydantic talk and thought I'd check it out. Seems like a fantastic idea but on openai==1.1.0 and instructor==0.3.0 raises a TypeError. This of course does not arise when using the "unpatched" openai client and sending the request, without the response_model kwarg

```
user = client.chat.completions.create(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'classmethod' object is not callable
```
thanks! and great talk

**Comment by morgann-instalily** [comment]
^^^

also 
"from openai import OpenAI()" 
should not have () that's invalid syntax

**Comment by imbesci** [comment]
Yes, I saw that typo too and handled it before trying it out!

**Comment by mkorpela** [comment]
Same

**Comment by kietoljw** [comment]
You can modify the code in the **patch.py** file as follows: 1. Change 'return classmethod(wrapper_function)' to 'return wrapper_function'; 2. Change 'model = response_model.from_response(response, validation_context=validation_context, strict=strict)' to 'model = response_model.from_response(response.model_dump(), validation_context=validation_context, strict=strict)'.

**Comment by kietoljw** [comment]
You can modify the code in the **patch.py** file to solve it as follows: 
1. Change `return classmethod(wrapper_function)` to  `return wrapper_function`; 
2. Change `model = response_model.from_response(response, validation_context=validation_context, strict=strict)` to `model = response_model.from_response(response.model_dump(), validation_context=validation_context, strict=strict)`

**Comment by cutmasta-kun** [comment]
I get the exact same error as @imbesci. When will this bug get fixed?

**Comment by jxnl** [comment]
sorry guys! working on this now!

**Comment by jxnl** [comment]
Just released 0.3.1, sorry guys! I realised i had turned off some tests I should not have.

**Comment by imbesci** [comment]
No worries, thought I was going crazy. Happy to raise it so its resolved!
